### DISCUSSION TITLE
Quick Malware Analysis: malware-traffic-analysis.net Hancitor pcap from 2021-05-20

### DISCUSSION THREAD

**Discussion by dougburks** [Security-Onion-Solutions/securityonion#5172] [Quick Malware Analysis]
Today's quick malware analysis with Security Onion!

Thanks to Brad Duncan for sharing this Hancitor pcap from 2021-05-20!

https://blog.securityonion.net/2021/08/quick-malware-analysis-malware-traffic_17.html

![Screen Shot 2021-08-13 at 10 59 05 AM](https://user-images.githubusercontent.com/1659467/129710324-ad80e5fa-e208-49ad-90b4-aaad591f9e2d.png)

![Screen Shot 2021-08-13 at 11 00 47 AM](https://user-images.githubusercontent.com/1659467/129710326-cdd56974-d9dc-4e61-98cc-4f5d1ae3e27b.png)

![Screen Shot 2021-08-13 at 11 01 18 AM](https://user-images.githubusercontent.com/1659467/129710329-4c600a7e-a20e-457d-9611-9f879fbd2765.png)

![Screen Shot 2021-08-13 at 11 05 03 AM](https://user-images.githubusercontent.com/1659467/129710331-9450a7b7-e6cb-43aa-98b5-baa641cc70de.png)

![Screen Shot 2021-08-13 at 11 05 07 AM](https://user-images.githubusercontent.com/1659467/129710333-add2481d-4d00-4b34-a4a0-ff0760a31350.png)

![Screen Shot 2021-08-13 at 11 06 35 AM](https://user-images.githubusercontent.com/1659467/129710336-b36ccc42-95d0-4beb-8d30-10d7f86aa514.png)

![Screen Shot 2021-08-13 at 11 07 23 AM](https://user-images.githubusercontent.com/1659467/129710338-176fdbd1-c024-46f8-9c96-6accbb41c949.png)

![Screen Shot 2021-08-13 at 11 07 35 AM](https://user-images.githubusercontent.com/1659467/129710339-4eeb995e-0474-40f7-a0fe-1fd12f4a5fbb.png)

![Screen Shot 2021-08-13 at 11 07 45 AM](https://user-images.githubusercontent.com/1659467/129710340-9ccb3b6b-102f-46ff-9611-b9f5cd5af739.png)

![Screen Shot 2021-08-13 at 11 10 06 AM](https://user-images.githubusercontent.com/1659467/129710341-a139978a-ea07-43f0-868d-10787dd931db.png)

![Screen Shot 2021-08-13 at 11 10 32 AM](https://user-images.githubusercontent.com/1659467/129710343-9b0ba880-391a-443c-b99a-6d983f7bcb0e.png)

**Comment by joeb1kenobe** [comment]
Doug,

I have been watching these. They are great. I am thinking of taking what
you are doing here and integrating that level of detail into or incident
response walkthroughs.

Thank you and the SO team for all the work you put into this great product!

Joe

On Tue, Aug 17, 2021 at 6:29 AM Doug Burks ***@***.***> wrote:

> Today's quick malware analysis with #SecurityOnion!
>
> Thanks to Brad Duncan for sharing this Hancitor pcap from 2021-05-20!
>
>
> https://blog.securityonion.net/2021/08/quick-malware-analysis-malware-traffic_17.html
>
> [image: Screen Shot 2021-08-13 at 10 59 05 AM]
> <https://user-images.githubusercontent.com/1659467/129710324-ad80e5fa-e208-49ad-90b4-aaad591f9e2d.png>
>
> [image: Screen Shot 2021-08-13 at 11 00 47 AM]
> <https://user-images.githubusercontent.com/1659467/129710326-cdd56974-d9dc-4e61-98cc-4f5d1ae3e27b.png>
>
> [image: Screen Shot 2021-08-13 at 11 01 18 AM]
> <https://user-images.githubusercontent.com/1659467/129710329-4c600a7e-a20e-457d-9611-9f879fbd2765.png>
>
> [image: Screen Shot 2021-08-13 at 11 05 03 AM]
> <https://user-images.githubusercontent.com/1659467/129710331-9450a7b7-e6cb-43aa-98b5-baa641cc70de.png>
>
> [image: Screen Shot 2021-08-13 at 11 05 07 AM]
> <https://user-images.githubusercontent.com/1659467/129710333-add2481d-4d00-4b34-a4a0-ff0760a31350.png>
>
> [image: Screen Shot 2021-08-13 at 11 06 35 AM]
> <https://user-images.githubusercontent.com/1659467/129710336-b36ccc42-95d0-4beb-8d30-10d7f86aa514.png>
>
> [image: Screen Shot 2021-08-13 at 11 07 23 AM]
> <https://user-images.githubusercontent.com/1659467/129710338-176fdbd1-c024-46f8-9c96-6accbb41c949.png>
>
> [image: Screen Shot 2021-08-13 at 11 07 35 AM]
> <https://user-images.githubusercontent.com/1659467/129710339-4eeb995e-0474-40f7-a0fe-1fd12f4a5fbb.png>
>
> [image: Screen Shot 2021-08-13 at 11 07 45 AM]
> <https://user-images.githubusercontent.com/1659467/129710340-9ccb3b6b-102f-46ff-9611-b9f5cd5af739.png>
>
> [image: Screen Shot 2021-08-13 at 11 10 06 AM]
> <https://user-images.githubusercontent.com/1659467/129710341-a139978a-ea07-43f0-868d-10787dd931db.png>
>
> [image: Screen Shot 2021-08-13 at 11 10 32 AM]
> <https://user-images.githubusercontent.com/1659467/129710343-9b0ba880-391a-443c-b99a-6d983f7bcb0e.png>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/5172>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAD6W6TS7CPJ46CTCNFRO3DT5I2Z3ANCNFSM5CJR2WOA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>
-- 
Joe Brown

**Reply by dougburks** [reply]
Thanks @joeb1kenobe !
